README.md

```diff
@@ -35,19 +40,11 @@ By default report is saved as *openvas.report*.
 #### Formats
 
 1. Anonymous XML
-2. ARF
```

Comment:
I see that multiple report formats were removed.
That may cause issues for people that use this container in an automatic scan and report pipeline.
I guess I'll have to make a legacy branch for OpenVAS 10, same as for OpenVAS 9 with HTML format.

Reply:
Yes, also some of the predefined port lists were removed too. Besides, port list id or port/ports need to be explicitly passed to the scanner now.

Reply:
By Greenbone documentatioon page, ARF is supported by OpenVAS 20.08: https://docs.greenbone.net/GSM-Manual/gos-20.08/en/reports.html#configuring-and-managing-report-formats

Is there a reason I am missing you removed it or is it just an oversight?

Reply:
It seems that their documentation might be out of date. More on that can be read here:
https://community.greenbone.net/t/gvm-20-08-missing-report-formats-and-scan-configs/6397/6